src/dochandle.rs

```diff
@@ -30,6 +32,7 @@ pub struct DocHandle {
     /// that doesn't require a mutabale reference to the handle.
     /// Note: the mutex is not shared between clones of the same handle.
     last_heads: Mutex<Vec<ChangeHash>>,
+    _not_sync: PhantomData<Cell<&'static ()>>,
```

Comment:
ref: https://doc.rust-lang.org/nomicon/phantom-data.html#table-of-phantomdata-patterns